KK-DZ/src/app/modules/workers/components/workers-table/workers-table.component.html

```diff
@@ -0,0 +1,34 @@
+<mat-table class="workers-table mat-elevation-z8" [dataSource]="workers" *ngIf="workers.length !== 0">
+  <ng-container matColumnDef="name">
+    <mat-header-cell *matHeaderCellDef>Nazwa</mat-header-cell>
+    <mat-cell *matCellDef="let worker">{{ worker.name }}</mat-cell>
+  </ng-container>
+  
+  <ng-container matColumnDef="age">
+    <mat-header-cell *matHeaderCellDef>Wiek</mat-header-cell>
+    <mat-cell *matCellDef="let worker">{{ worker.age }}</mat-cell>
+  </ng-container>
+  
+  <ng-container matColumnDef="city">
+    <mat-header-cell *matHeaderCellDef>Miasto</mat-header-cell>
+    <mat-cell *matCellDef="let worker">{{ worker.city }}</mat-cell>
+  </ng-container>
+  
+  <ng-container matColumnDef="isWorking">
+    <mat-header-cell *matHeaderCellDef>Status pracy</mat-header-cell>
+    <mat-cell *matCellDef="let worker">{{ worker.isWorking ? 'pracujący' : 'bezrobotny' }}</mat-cell>
```

Comment:
jest ok, ale jest to idealne miejsce żeby zastosować Angular pipe poczytajcie o tym https://angular.io/guide/pipes i stwórzcie swój pipe